library-model/test-library-model-generator/src/main/java/com/uber/nullaway/libmodel/AnnotationExample.java

```diff
@@ -0,0 +1,35 @@
+package com.uber.nullaway.libmodel;
+
+import java.util.Locale;
+
+/**
+ * This class has the same name as the class under
+ * resources/sample_annotated/src/com/uber/nullaway/libmodel/AnnotationExample.java because we use
+ * this as the unannotated version for our test cases to see if we are appropriately processing the
+ * annotations as an external library model.
```

Comment:
Technically, the way this will work in the actual usage is that the unannotated bytecode and the `astubx` file will be in two separate jars, no? Wonder if it's worth the effort to faithfully duplicate that setup in our integration tests...

Reply:
I think you're right, but I'm not sure it's worth trying to set this up here; it would be a bit of work (unless I'm missing a very simple way to do it).  I believe that in manual testing @akulk022 has already tried modeling actual JDK classes with this functionality and it seems to have worked, is that right?

Reply:
Yes I tested it with some methods from`java.util.function.Function` and `java.util.Vector` and that seemed to worked fine.